docs/V1_Plan.md

```
@@ -27,4 +27,7 @@ Version 1 serves as POC the project is interesting enough to continue.
 | ---- | ---- |  
 | erase | simply erases a local file over writing file with 0s before deleting it |  
 
+## Tech
```

Markdownlint

Multiple headings with the same content

(MD024, no-duplicate-heading)